onnxruntime/core/framework/orttraining_partial_executor.cc

```diff
@@ -233,6 +233,24 @@ Status PartialExecutor::Execute(const SessionState& session_state, const std::ve
     // construct OpKernelContext
     // TODO: log kernel inputs?
     OpKernelContextInternal op_kernel_context(session_state, frame, *p_op_kernel, logger, false);
+
+    // Cache lookup. Currently we only cache single-output nodes,
+    // to keep memory overhead impact in check. Hence we only look in cache
+    // if the current node has one output.
+    bool reuse_cached_value = false;
+    std::string cached_arg_name;
+#ifdef ENABLE_TRAINING
```

Comment:
Not needed, this whole file is under this ifdef, please see the first line of the file :-)